src/sentry/services/sentryshot/service.py

```diff
@@ -0,0 +1,48 @@
+import logging
+
+from selenium import webdriver
+from selenium.webdriver.chrome.options import Options
+from selenium.webdriver.common.by import By
+from selenium.webdriver.support import expected_conditions as EC
+from selenium.webdriver.support.ui import WebDriverWait
+
+from sentry.models.sentryshot import SentryShot
+
+MAX_WAIT_TIME = 5
+logger = logging.getLogger("service:sentryshot")
+
+
+class SentryShotService:
+    def __init__(self, uuid):
+        self.sentryshot = SentryShot.objects.get(uuid=uuid)
+
+    def _start(self):
+        chrome_options = Options()
+        chrome_options.add_argument("--headless")
+        self.driver = webdriver.Chrome(options=chrome_options)
+
+    def generate_screenshot(self):
+        # TODO: @athena check gcs first to see if the image is already generated
+        try:
+            self._start()
+            # TODO: @athena sign the url so it can access all sentry orgs
+            self.driver.get(self.sentryshot.sentry_url)
+            component_element = WebDriverWait(self.driver, MAX_WAIT_TIME).until(
+                EC.presence_of_element_located(
+                    (By.CLASS_NAME, self.sentryshot.component_identifier)
+                )
+            )
+            screenshot_path = "screenshot.png"
+            component_element.screenshot(screenshot_path)
```

Comment:
This is just for showing the code works. Will remove it before merging the PR.

In example of above screenshot I created the sentryshot object like this:
```
sentryshot = SentryShot.objects.create(sentry_url="http://127.0.0.1:8000/auth/login/sentry/", component_identifier="content", organization_id=1)
```